### PR TITLE
Misc_cleanup

### PR DESCRIPTION
**Pull request by PikalaxALT**
Additionally merges #275, #286

### CODE REVIEW

**Comment by adrienntindall** [comment]
Heads up you have tons of merge conflicts you'll have to work through soon

**Comment by adrienntindall** [comment]
Address conflicting files

File: CMakeLists.txt [hunk 1]
```diff
@@ -16,11 +16,11 @@ SET(GAME_ASM_DIR ${CMAKE_SOURCE_DIR}/asm/)
 SET(LIBS_SRC_DIR ${CMAKE_SOURCE_DIR}/lib/)
 SET(LIBS_ASM_DIR ${CMAKE_SOURCE_DIR}/lib/)
 file(GLOB_RECURSE C_SOURCES_GAME "${GAME_SRC_DIR}/*.c")
-file(GLOB_RECURSE C_SOURCES_LIBS "${LIBS_SRC_DIR}/*.c")
-file(GLOB_RECURSE CXX_SOURCES_LIBS "${LIBS_SRC_DIR}/*.cpp")
+# file(GLOB_RECURSE C_SOURCES_LIBS "${LIBS_SRC_DIR}/*.c")
```
Comment (adrienntindall) [inline CMakeLists.txt:19]: should we just delete these lines if we're gonna comment them out anyways?

File: src/scrcmd_battle_castle.c [hunk 2]
```diff
@@ -77,7 +77,7 @@ BOOL ScrCmd_637(ScriptContext *ctx) {
         case 3: // Unused
             sub_0204FA14(ctx->fieldSystem->saveData, unk0, arg1);
             break;
-        case 4:
+        case 4: {
```
Comment (adrienntindall) [inline src/scrcmd_battle_castle.c:80]: move bracket to new line

File: src/scrcmd_mystery_gift.c [hunk 3]
```diff
@@ -153,11 +153,12 @@ BOOL ScrCmd_MysteryGift(ScriptContext *ctx) {
         *ptr = pFunc->check(ctx->fieldSystem, FieldSystem_GetDataOfNextMG(ctx->fieldSystem));
         break;
     }
-    case SCR_MG_RECEIVE:
+    case SCR_MG_RECEIVE: {
```
Comment (adrienntindall) [inline src/scrcmd_mystery_gift.c:156]: move bracket to new line

File: src/unk_0204A3F4.c [hunk 4]
```diff
@@ -589,7 +589,7 @@ u32 sub_0204B120(UnkStruct_Fsys_A0 *a0, SaveData *saveData, u32 a2) {
             }
             return 1;
         }
-        case 2:
+        case 2: {
```
Comment (adrienntindall) [inline src/unk_0204A3F4.c:592]: move bracket to new line

File: src/voltorb_flip/voltorb_flip.c [hunk 5]
```diff
@@ -964,7 +980,8 @@ BOOL ov122_021E6900(WorkflowEngine *workflow, VoltorbFlipAppWork *work) {
 BOOL AwaitQuitYesNoSelection(WorkflowEngine *workflow, VoltorbFlipAppWork *work) {
     int var1 = YesNoPrompt_HandleInput(work->unk13C);
     switch (var1) {
-    case 1: // YES
+    case YESNORESPONSE_YES: // YES
```
Comment (adrienntindall) [inline src/voltorb_flip/voltorb_flip.c:983]: remove comment

File: src/voltorb_flip/voltorb_flip.c [hunk 6]
```diff
@@ -976,7 +993,8 @@ BOOL AwaitQuitYesNoSelection(WorkflowEngine *workflow, VoltorbFlipAppWork *work)
             EnqueueWorkflow(workflow, WORKFLOW_AWARD_COINS);
         }
         return TRUE;
-    case 2: // NO
+    }
+    case YESNORESPONSE_NO: // NO
```
Comment (adrienntindall) [inline src/voltorb_flip/voltorb_flip.c:997]: remove comment

**Review by adrienntindall — CHANGES_REQUESTED** [review]

File: include/voltorb_flip/voltorb_flip_data.h [hunk 7]
```diff
@@ -25,8 +25,8 @@ typedef struct BgTemplates {
     BgTemplate unk0[6];
 } BgTemplates;
 
-const u8 sMainMenuMsgNos[];
-const u8 sGameInfoMsgNos[];
-const u8 ov122_021E9270[];
+extern const u8 sMainMenuMsgNos[];
```
Comment (adrienntindall) [inline include/voltorb_flip/voltorb_flip_data.h:28]: do these need to be defined in the header or can we move them to the c file?

**Review by adrienntindall — CHANGES_REQUESTED** [review]

File: include/voltorb_flip/voltorb_flip_data.h (hunk 7 above)
Comment (PikalaxALT) [reply include/voltorb_flip/voltorb_flip_data.h:28]: Splitting voltorb_flip.c and merging them with the respective voltorb_flip_data*.c is out-of-scope for this PR.

**Review by adrienntindall — APPROVED** [review]

**Pull request merged**